pkg/trtl/testdata/db.json

```diff
@@ -94,15 +94,5 @@
         "value": {
             "data": "some unclassified data"
         }
-    },
-    "peer": {
-        "namespace": "peers",
-        "key": "82abc34d-f8f9-4f0e-b8e3-f8f9f8f8f8f8",
-        "value": {
-            "Id": 1,
-            "Addr": "127.0.0.1:1234",
-            "Name": "peer",
-            "Region": "us-east-1"
-        }
```

Comment:
The peer needs to be in protocol buffer wire format, so we can't add it as JSON to the fixtures unfortunately.